perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ExtSourcesManagerBlImpl.java

```diff
@@ -281,7 +281,7 @@ public Candidate getCandidate(PerunSession sess, ExtSource source, String login)
 						try {
 							// Create new one if not exists
 							additionalExtSource = new ExtSource(additionalExtSourceName, additionalExtSourceType);
-							additionalExtSource = getPerunBl().getExtSourcesManagerBl().createExtSource(sess, additionalExtSource);
+							additionalExtSource = getPerunBl().getExtSourcesManagerBl().createExtSource(sess, additionalExtSource, attributes);
```

Comment:
These attributes are not extSource attributes. They are attributes of new candidate. There must be null.